### PR TITLE
Issue 42: expose additional visualizations generated by QUAST in the evaluate-contigs visualization

### LINKED ISSUE #42: expose additional visualizations generated by QUAST in the `evaluate-contigs` visualization
It seems that QUAST generates many more visualizations than the ones that are currently displayed in the visualization produced by `evaluate-contigs` (most, if not all, of them are generated based on alignments to reference sequences, either provided by the user - not yet supported, see #35 - or fetched by QUAST automatically). Some of the interesting ones include:
- Krona plots per sample
- more Icarus reports (per reference sequence - show coverage per reference genome in the contig browser)
- reports for not aligned sequences

### PR DESCRIPTION
**Pull request by Sann5**
Closes #42: "expose additional visualizations generated by QUAST in the evaluate-contigs visualization"

- Adds a button in the QC Report tab which downloads additional reports.
- Adds a tab showing the Krona plots with a drop-down menu to choose between samples and summary. It also deletes the hyperlinks previously available in the QC Report tabb to visualize the Krona plots.

### CODE REVIEW

**Comment by Sann5** [comment]
### TODO 🗒️

- [ ] Add test for new functions
   - [`_zip_additional_reports`](https://github.com/Sann5/q2-assembly/blob/83ab8946daaf26758489563d5f0d4efd72731fae/q2_assembly/quast/quast.py#L269)
   - [`_zip_dir`](https://github.com/Sann5/q2-assembly/blob/83ab8946daaf26758489563d5f0d4efd72731fae/q2_assembly/quast/quast.py#L196)
   - [`_zip_additional_reports`](https://github.com/Sann5/q2-assembly/blob/83ab8946daaf26758489563d5f0d4efd72731fae/q2_assembly/quast/quast.py#L204)

File: q2_assembly/quast/tests/test_quast.py [hunk 1]
```diff
@@ -486,11 +492,90 @@ def test_evaluate_contigs_action_paired_end(self, p1, p2, p3, p4):
...
+            for i in range(3) for j in range(3) for k in range(3)
+        ]
+
+        all_files = [*folder1_files, *folder2_files, *folder3_files]
+
+        for file in all_files:
+            try:
+                open(file, 'a').close()
+            except Exception:
+                os.makedirs(file)
```
Comment (misialq) [inline q2_assembly/quast/tests/test_quast.py:522]: Please just directly create all those files in the `tests/data` directory - no need for another function. They will be empty anyways so no space waste and they will simplify the code greatly.

**Review by misialq — CHANGES_REQUESTED** [review]
Hey @Sann5, looks great! I tried the viz and all looks as expected - thanks! 🥇 Just one comment below.

File: setup.py [hunk 2]
```diff
@@ -32,6 +32,19 @@
...
+            "data/zip_test_data/expected/folder_1/subfolder_2/*",
+            "data/zip_test_data/expected/folder_2/subfolder_0/subsubfolder_0/*",
+            "data/zip_test_data/expected/folder_2/subfolder_0/subsubfolder_1/*",
+            "data/zip_test_data/expected/folder_2/subfolder_0/subsubfolder_2/*",
+            "data/zip_test_data/expected/folder_2/subfolder_1/subsubfolder_0/*",
+            "data/zip_test_data/expected/folder_2/subfolder_1/subsubfolder_1/*",
+            "data/zip_test_data/expected/folder_2/subfolder_1/subsubfolder_2/*",
+            "data/zip_test_data/expected/folder_2/subfolder_2/subsubfolder_0/*",
+            "data/zip_test_data/expected/folder_2/subfolder_2/subsubfolder_1/*",
+            "data/zip_test_data/expected/folder_2/subfolder_2/subsubfolder_2/*",
```
Comment (misialq) [inline setup.py:47]: You could replace it with (you'll only really know if it worked properly after the CI is finished):
```suggestion
            "data/zip_test_data/expected/*",
            "data/zip_test_data/expected/*/*",
            "data/zip_test_data/expected/*/*/*",
            "data/zip_test_data/expected/*/*/*/*",
```

**Pull request merged**